### DISCUSSION TITLE
Make object selectable/movable under lines grid.

### DISCUSSION THREAD

**Discussion by KhorenGrig** [fabricjs/fabric.js#7480] [Q&A] [answered]
I have drawn a cell/grid on the canvas(in this way). 

![code](https://user-images.githubusercontent.com/26667922/140807420-410b7963-65d2-43f1-873e-cafe17bc0a90.png)

The question is how to keep this grid always on the top of the canvas, but at the same time make existing objects(text layers, custom elements, etc.) under this grid movable/selectable.

P.S. I am new in Fabric js and do not know lib well, MY GOAL IS to draw grid/cell on canvas(anytime) and make other objects (which were added on canvas before activation of the grid) movable/selectable under that grid. The grid should be toggled.

**Comment by asturur** [comment, marked as answer]
You should add this group as an canvas.overlayImage

canvas.overlayImage = group;

This will bring you one step further in that direction

**Reply by KhorenGrig** [reply]
Thanks a lot. It's worked.
